app/api/account_session/route.ts

```diff
@@ -152,6 +152,10 @@ export async function POST(req: NextRequest) {
         tax_registrations: {
           enabled: true,
         },
+        // @ts-ignore
```

Comment:
@kevinpeters-stripe have we not shipped these components ^ to the SDK? Can we remove these ts-ignore?

@cmochrie-stripe Can we please add some context to the ts-ignore here?
```suggestion
        // This component isn't yet in the SDK, but once it is, we can remove this
        // @ts-ignore
```

Reply:
Ok